
 make.py started: 2016-08-01 11:50:52 /Users/Frank/Desktop/gslab_python/eatthepie 


Path: gslab_make
URL: https://econ-gentzkow-svn.stanford.edu/repos/main/trunk/lib/python/gslab_make
Repository Root: https://econ-gentzkow-svn.stanford.edu/repos/main
Repository UUID: 89b4331b-4301-411a-8e06-9c78d9cdb83d
Revision: 34785
Node Kind: directory
Last Changed Author: jwchoi2
Last Changed Rev: 33502
Last Changed Date: 2016-02-07 17:40:17 -0800 (Sun, 07 Feb 2016)



Execute:  python  "run_all_tests.py" 
test_eatthepie (unittest.loader.ModuleImportFailure) ... ERROR
test_neg (test_MiscFunctions.test_approxloc) ... ok
test_outofrange (test_MiscFunctions.test_approxloc) ... ok
test_pos (test_MiscFunctions.test_approxloc) ... ok

======================================================================
ERROR: test_eatthepie (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_eatthepie
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/Frank/Desktop/gslab_python/eatthepie/test/test_eatthepie.py", line 6, in <module>
    import py_module.eatthepie as eatthepie
  File "../py_module/eatthepie.py", line 7, in <module>
    import ConsumptionModel as CM
  File "../py_module/ConsumptionModel.py", line 23, in <module>
    from numba import jit
ImportError: No module named numba


----------------------------------------------------------------------
Ran 4 tests in 0.000s

FAILED (errors=1)

 make.py ended: 2016-08-01 11:50:54
